src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -535,6 +562,29 @@ public override void ProjectStartedHandler(object sender, ProjectStartedEventArg
                     WriteItems(e, e.Items);
                 }
             }
+
+            
+            string target_framework = "";
+
+            foreach (KeyValuePair<string, string> prop in e.GlobalProperties)
+            {
+                if ((string)prop.Key == "TargetFramework")
+                {
+                    target_framework = (string)prop.Value;
+                }
```

Comment:
Add break;
?

Reply:
Good point. I changed it to use the TryGetValue() instead